docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -30,6 +30,39 @@ description: >
 * When upgrading to a new minor version, a new OS image must be created using the new image-builder CLI pertaining to that release.
 {{% /alert %}}
 
+## [v0.20.3](https://github.com/aws/eks-anywhere/releases/tag/v0.20.3)
+
+### Supported OS version details
+|                     | vSphere | Bare Metal | Nutanix | CloudStack | Snow |
+|:-------------------:|:-------:|:----------:|:-------:|:----------:|:----:|
+|    Ubuntu 20.04     |    ✔    |     ✔      |    ✔    |     —      |  ✔   |
+|    Ubuntu 22.04     |    ✔    |     ✔      |    ✔    |     —      |  —   |
+| Bottlerocket 1.20.0 |    ✔    |     —      |    —    |     —      |  —   |
+| Bottlerocket 1.19.4 |    —    |     \*      |    —    |     —      |  —   |
+|      RHEL 8.x       |    ✔    |     ✔      |    ✔    |     ✔      |  —   |
+|      RHEL 9.x       |    —    |     ✔      |    ✔    |     ✔      |  —   |
+
+\* [EKS Anywhere issue regarding deprecation of Bottlerocket bare metal variants](https://github.com/aws/eks-anywhere/issues/7754)
+
+### Changed
+- EKS Distro:
+  - `v1-27-eks-35` to [`v1-27-eks-36`](https://distro.eks.amazonaws.com/releases/1-27/36/)
+  - `v1-28-eks-28` to [`v1-28-eks-29`](https://distro.eks.amazonaws.com/releases/1-28/29/)
+  - `v1-29-eks-17` to [`v1-29-eks-18`](https://distro.eks.amazonaws.com/releases/1-29/18/)
+  - `v1-30-eks-10`  to [`v1-30-eks-11`](https://distro.eks.amazonaws.com/releases/1-30/11/)
+
+### Added
+- Enable Tinkerbell stack to use dhcrelay instead of using smee in hostNetwork mode. ([#8568](https://github.com/aws/eks-anywhere/pull/8568))
```

Comment:
```suggestion
- Enable Tinkerbell stack to use dhcprelay instead of using smee in hostNetwork mode. ([#8568](https://github.com/aws/eks-anywhere/pull/8568))
```